packages/amazon-sumerian-hosts-babylon/examples/common/demo-utils.js

```diff
@@ -0,0 +1,41 @@
+function setupSceneEnvironment(scene) {
+  // Create a simple environment.
+  const environmentHelper = scene.createDefaultEnvironment({
+    groundOpacity: 1,
+    groundShadowLevel: 0.3
+  });
+  environmentHelper.setMainColor(BABYLON.Color3.Teal());
+
+  // const light = new BABYLON.HemisphericLight("light", new BABYLON.Vector3(-0.3, 1, 0));
```

Comment:
nit: do we want to remove the comment out line

Reply:
Thanks for catching this. I will fix.

Reply:
This issue is addressed in the update I just posted.